test/dotcom/trip_plan/open_street_map_reconciler_test.exs

```diff
@@ -0,0 +1,82 @@
+defmodule Dotcom.TripPlan.OpenStreetMapReconcilerTest do
+  use ExUnit.Case
+
+  import Dotcom.TripPlan.OpenStreetMapReconciler, only: [reconcile: 1, reconcile_location: 1]
+
+  alias Dotcom.TripPlan.OpenStreetMapReconciler
+  alias Test.Support.Factories.TripPlanner.TripPlanner
+
+  @location_inside_state_house %{latitude: 42.35878, longitude: -71.063812}
```

Comment:
Didn't use Faker for this because I couldn't figure out a way to do this without having some hard-coded values, and wound up in a kind of complicated mess when I tried. 

I'm open to ideas and especially open to pairing on a solution here, although I want to balance that against the fact that I've already spent a lot of time on this, and it would be good to get the rider-facing bugfix out there sooner rather than later.